headers/types/dungeon_mode/dungeon.h

```diff
@@ -1626,12 +1625,13 @@ struct dungeon {
     // 0x2B332: Spawn weights for bazaar grab bag items. Same format as regular_item_weights.
     uint16_t grab_bag_item_weights[1416];
     // 0x2BE42: Spawn weights for secret room items in treasure boxes.
-    // Same format as regular_item_weights.
+    // Same format as regular_item_weights? For some reason the weights for the secret rooms
+    // are of a different length than the other item weight lists before it? The trap weights
+    // appear to overlap what would be the 16 categories entries? Maybe categories not used for
+    // this list?
```

Comment:
> The trap weights appear to overlap what would be the 16 categories entries?

Not true, category weights are stored at the start of the list, the trap data overlaps with the end of the list.
Also, the code does write category weights here, so that can't be the reason.

```suggestion
    // Same format as regular_item_weights? For some reason the weights for the secret rooms
    // are of a different length than the other item weight lists before it. The trap weights
    // appear to overlap what would be entries 1400-1415.
```